README.md

```diff
@@ -1,25 +1,33 @@
 # bem-twig-extension
-Twig function that inserts static classes into Pattern Lab but adds them to the attributes object in Drupal
+Twig function that inserts static classes into Pattern Lab but adds them to the Attributes object in Drupal
 
-## Usage
+## Usage (4 arguments)
 
-#### Simple block name:
-`<h1{{ bem('title') }}>`
+#### Simple block name (required argument):
+`<h1 {{ bem('title') }}>`
 
 This creates:
 
 `<h1 class="title">`
 
-#### Block with modifiers (array allowing multiple modifiers):
-`<h1{{ bem('title', ({1: 'small', 2: 'red'})) }}>`
+#### Block with modifiers (optional array allowing multiple modifiers):
+`<h1 {{ bem('title', (['small', 'red'])) }}>`
```

Comment:
I don't think you need the parentheses, so `(['small', 'red'])` could just be `['small', 'red']`